test/spec/estimatedScore/estimatedScoreEventsHandlerSrv.spec.js

```diff
@@ -134,7 +133,7 @@ describe('testing service "EstimatedScoreEventsHandlerSrv":', function () {
     });
 
     it('when drill exercise is completed then raw score and estimated should be updated accordingly', function () {
```

Comment:
when drill...if changed to section change also here the description of this test